Dockerfile

```diff
@@ -9,27 +9,17 @@
 FROM mikefarah/yq as builder
 RUN apk add --no-cache bash
 COPY .htaccess README.md ./scripts/*.sh /build/
-COPY /plugins /build/plugins
-COPY /v2 /build/v2
 COPY /v3 /build/v3
 WORKDIR /build/
-RUN ./check_plugins_location_v1.sh
-RUN ./check_plugins_location_v2.sh v2
 RUN ./check_plugins_location_v2.sh v3
 RUN ./check_plugins_images.sh
 RUN ./set_plugin_dates.sh
-RUN ./check_plugins_viewer_mandatory_fields_v1.sh
-RUN ./check_plugins_viewer_mandatory_fields_v2.sh
 RUN ./check_plugins_viewer_mandatory_fields_v3.sh
 RUN ./ensure_latest_exists.sh
-RUN ./index.sh > /build/plugins/index.json
-RUN ./index_v2.sh v2 > /build/v2/plugins/index.json
 RUN ./index_v2.sh v3 > /build/v3/plugins/index.json
```

Comment:
We could rename this to `index.sh` or `index_v3.sh` for consistency (since `v2` is gone as of this PR).